app/models/booking/booking.ts

```diff
@@ -23,7 +23,7 @@ export class Booking extends RestModel implements Updateable, Createable{
   private _getRelevantData(): Object {
     return {
       workpackage: this.workpackage.self.toString(),
-      date: this.date,
+      date: this.date.format('YYYY-MM-DD'),
       effort: this.effort,
```

Comment:
This is a very important change we have to take care of not to be overwritten by old code on developement branch!